### PR TITLE
docs: documentation on resilient streaming consumer fixed

### PR DESCRIPTION
**Pull request by augi**

### CODE REVIEW

**Review by sideeffffect — APPROVED** [review]
Good catch!

File: README.md [hunk 1]
```diff
@@ -155,9 +155,9 @@ val deliveryStream: Resource[Task, fs2.Stream[Task, StreamedResult]] = for {
     val stream: fs2.Stream[Task, StreamedResult] = streamingConsumer.deliveryStream.through(processMyStream)
     
     // create resilient (self-restarting) stream; see more information below
-    val resilientStream: fs2.Stream[Task, StreamedResult] = stream.handleErrorWith { _ =>
+    val resilientStream: fs2.Stream[Task, StreamedResult] = stream.handleErrorWith { e =>
```
Comment (sideeffffect) [inline README.md:158]: why not keep `_`?

**Review by sideeffffect — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (augi) [reply README.md:158]: To show that there is an exception that should be processed somehow.

**Review by jendakol — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jendakol) [reply README.md]: While I think it's obvious the error is there, I have no problem with this.

**Pull request merged**